src/app/EventManagement.cpp

```diff
@@ -583,31 +613,46 @@ CHIP_ERROR EventManagement::CopyEvent(const TLVReader & aReader, TLVWriter & aWr
 {
     TLVReader reader;
     TLVType containerType;
+    TLVType containerType1;
+    TLVType containerType2;
```

Comment:
It looks the containerType1 and containerType2 are used one after another, could we just reuse?

Reply:
yep, done, thanks